src/main/java/dev/andrylat/carsharing/dao/BodyTypeDAO.java

```diff
@@ -0,0 +1,40 @@
+package dev.andrylat.carsharing.dao;
+
+import dev.andrylat.carsharing.models.BodyType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class BodyTypeDAO {
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public BodyTypeDAO(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public List<BodyType> showAll() {
+        return jdbcTemplate.query("SELECT * FROM body_types", new BodyTypeMapper());
+    }
+
+    public BodyType showById(int id) {
+        return jdbcTemplate.query("SELECT * FROM body_types WHERE id=?", new BodyTypeMapper(), id)
+                .stream().findAny().orElse(null);
+    }
+
+    public void add(BodyType bodyType) {
```

Comment:
You will add new body type. How will you receive inserted data? for example, you should send added object to the front to show user this data and give him possibility to remove or update data.